Formula/arangodb.rb

```diff
@@ -5,6 +5,8 @@ class Arangodb < Formula
   sha256 "04c00d58d7e63137ccb7d0a73112aa01e34e08c98b3c82c62ef7987f0d214ac2"
   head "https://github.com/arangodb/arangodb.git", :branch => "unstable"
 
+  revision 1
```

Comment:
This should go between the `sha256` and `head` line above, with an empty line between the `revision` and the `head` line.